pkg/apis/sessions/interfaces.go

```diff
@@ -10,3 +13,12 @@ type SessionStore interface {
 	Load(req *http.Request) (*SessionState, error)
 	Clear(rw http.ResponseWriter, req *http.Request) error
 }
+
+var ErrLockNotObtained = fmt.Errorf("lock: not obtained")
```

Comment:
We should probably use the errors package for this
```suggestion
var ErrLockNotObtained = errors.New("lock: not obtained")
```